firedrake/interpolation.py

```diff
@@ -480,6 +495,29 @@ def compose_map_and_cache(map1, map2):
     return cmap
 
 
+def vom_cell_parent_node_map_extruded(vertex_only_mesh, extruded_cell_node_map):
```

Comment:
I think more documentation here describing why this is needed and how it works would be useful.